Fabric.Terminology.SqlServer/Persistence/SqlValueSetRepository.cs

```diff
@@ -129,15 +129,15 @@ public IValueSet GetValueSet(string valueSetId, IEnumerable<string> codeSystemCo
         }
 
         public Task<PagedCollection<IValueSet>> FindValueSetsAsync(
-            string nameFilterText,
+            string filterText,
             IPagerSettings pagerSettings,
             IEnumerable<string> codeSystemCodes,
             bool includeAllValueSetCodes = false)
         {
             var dtos = this.DbSet.Where(GetBaseExpression());
-            if (!nameFilterText.IsNullOrWhiteSpace())
+            if (!filterText.IsNullOrWhiteSpace())
             {
-                dtos = dtos.Where(dto => dto.ValueSetNM.Contains(nameFilterText));
+                dtos = dtos.Where(dto => dto.ValueSetNM.Contains(filterText) || dto.ValueSetOID.Contains(filterText));
```

Comment:
In my opinion, the `dto.ValueSetOID.Contains(filterText)` is pretty loose and will probably muddle up other legit queries.  I'm leaning toward changing this to `dto.ValueSetOID == filterText` e.g. so the name can contain by the ID would need to equal.

Thoughts?